bootcamp_project/Module 2/Guessing_Game_Guard_Clause/Guessing_Game_Guard_Clause_EMira1.ts

```diff
@@ -0,0 +1,45 @@
+const promptSync = require('prompt-sync')({sigint: true});
+const myUserName: string = promptSync("Hey there! What is your name? ");
+console.log("Welcome " + myUserName + " , A random number between 0 and 99 will be generated and you'll have 3 opportunities to guess it. Let's play!")
+
+
+const randomNumber: number = getRandom(0,99);
+let attempt: number = 3;
+
+
+
+     while (attempt > 0) {
```

Comment:
Nitpicky suggestion for code style:
```suggestion
let attempt: number = 3;

     while (attempt > 0) {
```

Reply:
Updated.